src/home-page/NoMatch404.tsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import { useHistory } from 'react-router-dom';
+import Typography from '@material-ui/core/Typography';
+import styled from 'styled-components';
+import { Button } from '@material-ui/core';
+import { MainContainer } from '../layout/components/MainContainer';
+
+const NoMatchTypography = styled('div')`
```

Comment:
I think this name here "NoMatchTypography" can be a bit confusing, because this component is more some kind of container, then Typography. In my opinion name Typography points more on some kind of pure text element. Maybe we can rename it to be more suitable and descriptive. Maybe 404Card or NoMatchCard, what do you think?

Reply:
I'd recommend simply a `CardWrapper` :)

Reply:
Done